main.h

```diff
@@ -34,6 +34,8 @@
 // Audio
 // fs: 50000 Hz
 #define PDM_BUFFER_LENGTH               (1000)
+#define PDM_DEC_FACTOR                  (4)
```

Comment:
let's avoid using abbreviations like DEC. Makes it difficult for new people who don't know that DEC maps to DECIMATION